2011-05-01  Michael Tautschnig  <mt@debian.org>

	* setup-storage.8: Document alignment strategy of setup-storage.
Index: trunk/man/setup-storage.8
===================================================================
--- trunk.orig/man/setup-storage.8
+++ trunk/man/setup-storage.8
@@ -696,6 +696,12 @@
 will want to copy these to some removable media. To make encrypted root devices
 actually usable, you need to add busybox (and initramfs-tools) to your package
 config.
+.IP \(bu
+For backwards compatibility or other system-specific reasons an alignment to
+cylinder boundaries may be necessary. Yet other systems will have other
+alignment constraints. setup-storage sets the alignment as follows: If align-at
+is set, align accordingly. Otherwise, if any partition on the particular disk is
+to be preserved, default to cylinder alignment. Else use sector alignment.
 .SH SEE ALSO
 This program is part of FAI (Fully Automatic Installation).
 The FAI homepage is http://fai-project.org.
